### PR TITLE
Adding NewsX API to News

### PR DESCRIPTION
**Pull request by unixguru2k**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1174,7 +1174,7 @@ API | Description | Auth | HTTPS | CORS |
 | [The Old Reader](https://github.com/theoldreader/api) | RSS reader | `apiKey` | Yes | Unknown |
 | [TheNews](https://thenewsapi.com/) | Aggregated headlines, top story and live news JSON API | `apiKey` | Yes | Yes |
 | [Trove](https://trove.nla.gov.au/about/create-something/using-api) | Search through the National Library of Australia collection of 1000s of digitised newspapers | `apiKey` | Yes | Unknown |
-
+| [NewsX by MACHAAO](https://rapidapi.com/machaao-inc-machaao-inc-default/api/newsx/) | Get or Search Latest Breaking News with ML Powered Summaries 🤖 | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:1177]: Remove this:  by MACHAAO

```suggestion
| [NewsX](https://rapidapi.com/machaao-inc-machaao-inc-default/api/newsx/) | Get or Search Latest Breaking News with ML Powered Summaries 🤖 | `apiKey` | Yes | Unknown |
```

And don't forget to fix the alphabetical order, please

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @unixguru2k ;)

Can you resolve this review, please?

And don't forget to fix the alphabetical order, please

**Comment by unixguru2k** [comment]
Thanks for guiding me @matheusfelipeog 
I have updated as per your suggestions.

Cheers
HD

**Comment by pawelborkar** [comment]
Thanks for the contribution @unixguru2k

**Pull request merged**